torch/csrc/jit/mobile/model_compatibility.cpp

```diff
@@ -179,6 +187,69 @@ std::unordered_map<std::string, OperatorInfo> _get_model_ops_and_info(
   return result;
 }
 
+/********************** Get Type Table **********************/
+
+// Forward declare
+std::unordered_set<std::string> _get_model_contained_types(
```

Comment:
Consider renaming it to `_get_**mobile**_model_contained_types`, because this function only get types from bytecode.